src/applications/personalization/profile360/vet360/actions/ui.js

```diff
@@ -1,159 +1,19 @@
-  const strippedExtension = (extension || '').replace(/[^a-zA-Z0-9]/g, '');
-
-  return {
-    id,
-    areaCode: strippedPhone.substring(0, 3),
-    countryCode,
-    extension: strippedExtension,
-    phoneType,
-    phoneNumber: strippedPhone.substring(3),
-    isInternational: countryCode !== '1',
-    inputPhoneNumber,
-  };
-}
-
-// @todo Consolidate this logic with consolidateAddress, which is intended to make address formats
-// consistent for display within the edit-address modal. Maybe they should go directly in that modal instead.
-// Also, should probably do something with the updaters' createAddressObject here too.
-export function cleanAddressDataForUpdate(value) {
-  const {
-    id,
-    addressLine1,
-    addressLine2,
-    addressLine3,
-    addressPou,
-    city,
-    countryName,
-    stateCode,
-    zipCode,
-    internationalPostalCode,
-    province,
-  } = value;
-
-  const addressType = inferAddressType(countryName, stateCode);
-
-  return {
-    id,
-    addressLine1,
-    addressLine2,
-    addressLine3,
-    addressPou,
-    addressType,
-    city,
-    countryName,
-    province: addressType === 'INTERNATIONAL' ? province : null,
-    stateCode: addressType === 'INTERNATIONAL' ? null : stateCode,
-    zipCode: addressType !== 'INTERNATIONAL' ? zipCode : null,
-    internationalPostalCode: addressType === 'INTERNATIONAL' ? internationalPostalCode : null,
-  };
-}
-
-function updateProfileFormField(field, validator, type) {
-  return (value, fieldName, skipValidation) => {
-    let cleanValue = value;
-
-    switch (type) {
-      case 'email':
-        cleanValue = cleanEmailDataForUpdate(value);
-        break;
-      case 'phone':
-        cleanValue = cleanPhoneDataForUpdate(value);
-        break;
-      case 'address':
-        cleanValue = cleanAddressDataForUpdate(value);
-        break;
-      default:
+    type: UPDATE_PROFILE_FORM_FIELD,
+    field: fieldName,
+    newState: {
+      value: cleanValue,
+      validations
     }
-
-    const validations = validator && !skipValidation ? validator(cleanValue, fieldName) : {};
-
-    return {
-      type: UPDATE_PROFILE_FORM_FIELD,
-      field,
-      newState: {
-        value: cleanValue,
-        validations
-      }
-    };
   };
 }
-
-export const updateFormField = {
-  email: updateProfileFormField('email', validateEmail, 'email'),
-  faxNumber: updateProfileFormField('faxNumber', validateTelephone, 'phone'),
-  homePhone: updateProfileFormField('homePhone', validateTelephone, 'phone'),
-  mailingAddress: updateProfileFormField('mailingAddress', validateAddress, 'address'),
-  mobilePhone: updateProfileFormField('mobilePhone', validateTelephone, 'phone'),
-  residentialAddress: updateProfileFormField('residentialAddress', validateAddress, 'address'),
-  temporaryPhone: updateProfileFormField('temporaryPhone', validateTelephone, 'phone'),
-  workPhone: updateProfileFormField('workPhone', validateTelephone, 'phone'),
-};
```

Comment:
The changes here are about the same as in `transactions.js`- action creators are now derived and the various formatters moved into the corresponding components.